sherpa/cpp_api/online-stream.h

```diff
@@ -149,6 +149,12 @@ class OnlineStream {
   // The returned reference is valid as long as this object is alive.
   int32_t &GetNumTrailingBlankFrames();
 
+  const int32_t GetFrameShift();
```

Comment:
```suggestion
  int32_t GetFrameShift() const;
```

Please also describe whether the returned value is in seconds or in milliseconds.